invenio_drafts_resources/records/api.py

```diff
@@ -90,9 +90,13 @@ class Record(RecordBase):
     versions = VersionsField(create=True, set_latest=True)
 
     @classmethod
-    def get_records_by_parent(cls, parent):
+    def get_records_by_parent(cls, parent, include_deleted=True):
```

Comment:
I guess is to be backwards compatible, but would we actually use this function with `include_deleted=True`?

Reply:
I don't know that, because the function is at the API level.
The `delete` of the draft, after publish, is happening at service level. So basically, this function doesn't know from where it will be called.